packages/generators/add-generator.ts

```diff
@@ -125,44 +143,46 @@ module.exports = class AddGenerator extends Generator {
 			.then(() => {
 				if (action === "entry") {
 					return this.prompt([
-						Confirm("entryType", "Will your application have multiple bundles?")
+						Confirm("entryType", "Will your application have multiple bundles?"),
 					])
-						.then(entryTypeAnswer => {
+						.then((entryTypeAnswer: {
+							entryType: boolean,
+						}) => {
 							// Ask different questions for entry points
 							return entryQuestions(self, entryTypeAnswer);
 						})
-						.then(entryOptions => {
+						.then((entryOptions: object) => {
 							this.configuration.config.webpackOptions.entry = entryOptions;
 							this.configuration.config.item = action;
 						});
 				}
-				let temp = action;
+				const temp: string = action;
```

Comment:
it's temp for a reason

Reply:
Okay, so what are you stating? Sorry, I don't understand.

Reply:
it's named temp for a reason, it get's mutated I assume?

Reply:
It isn't getting mutated. It would have anyway caused linting error if we were reassigning it.

Reply:
alright. Nice